site/content/docs/5.3/components/scrollspy.md

```diff
@@ -24,6 +24,9 @@ Scrollspy toggles the `.active` class on anchor (`<a>`) elements when the elemen
 
 ### Navbar
 
+<details>
+<summary>See Bootstrap example(s) that are incompatible with Orange Design System.</summary>
+<br>
```

Comment:
When embedded into the `<details><summary>` the example doesn't seem to work. Is there a way to make it work?